controller/controller.go

```diff
@@ -154,6 +159,16 @@ func New(config Config) (*Controller, error) {
 		}
 	}
 
+	sentryConfig := sentry.Config{
+		Dsn:    config.SentryDSN,
+		Logger: config.Logger,
+	}
+	sentryClient, err := sentry.New(sentryConfig)
+	if err != nil {
+		// This is not a blocking error, we just want to log it.
+		config.Logger.LogCtx(context.Background(), "level", "error", "message", "Error initializing Sentry client", "stack", microerror.JSON(err))
+	}
```

Comment:
It would be cool if this could be aligned with the rest of the code base. Meaning scope block and `c` variable name for config structures. I also do think that if the initiation fails the error should be returned. What is the reason why we should proceed controller initialization in case such an error occurs?

Reply:
I agree with the naming.
The goal here was not to block the operator from functioning in case sentry is not working.
Think about a disruption in their service: making our operator not to reconcile because of a third party service is down doesn't feel smart to me. WDYT?

Reply:
There are different levels of information we should provide or ignore. When creating a controller we should initialize what is configured. When nobody wants Sentry support we should not construct it. A user should not be bothered about information they do not have asked for. Like dropping a news message in a news channel telling you there are no news. Logging can then happen in the constructor of the feature implementation only, if at all.

Reply:
ok done